Makefile

```diff
@@ -1,10 +1,15 @@
+CC=clang
+CXX=clang++
+LLVM_CONFIG=llvm-config
+export
+
 all:
-	clang++ -glldb $(shell llvm-config --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions main.cpp -o ./main
+	$(CXX) -glldb $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions main.cpp -o ./main
```

Comment:
I *think* (but I might be very wrong!) that the `$(shell ...)` bit is adding what are already the defaults to the command?

Reply:
No no, the llvm-config thing is used to determine the flags needed to link against libLLVM with is different then the default clang flags.

Reply:
Ah OK!